google/resource_task_queue_test.go

```diff
@@ -0,0 +1,276 @@
+package google
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"golang.org/x/net/context"
+	"google.golang.org/api/cloudtasks/v2"
+)
+
+func TestAccTaskQueue_basic(t *testing.T) {
+	t.Parallel()
+
+	var queue cloudtasks.Queue
+	queueName := fmt.Sprintf("tf-test-task-queue-%d", acctest.RandInt())
+	location := "us-central1"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccTaskQueueDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTaskQueue_basic(queueName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckTaskQueueExists(
+						"google_task_queue.fizzbuzz", location, queueName, &queue),
+					testAccCheckTaskQueueEquals(
+						"google_task_queue.fizzbuzz", &queue),
+				),
```

Comment:
I don't think we need these checks (nor the funcs, and I think that also means we can avoid vendoring this new `context` dependency) - the `ImportStateVerify` is sufficient.